docs/components/input-controller/_index.md

```diff
@@ -44,281 +44,475 @@ Once you've configured your input controller according to model type, you can wr
 
-BUTTON_PRESS = "ButtonPress"
-"""
-Typical key press.
-"""
+**Returns:**
 
-BUTTON_RELEASE = "ButtonRelease"
-"""
-Key release.
-"""
+- `controls` [(List[Control])](https://python.viam.dev/autoapi/viam/components/input/index.html#viam.components.input.Control): List of Controls provided by the controller.
 
-BUTTON_HOLD = "ButtonHold"
-"""
-Key is held down. This will likely be a repeated event.
-"""
+For more information, see the [Python SDK Docs](https://python.viam.dev/_modules/viam/components/input/input.html#Controller.get_position).
 
-BUTTON_CHANGE = "ButtonChange"
-"""
-Both up and down for convenience during registration, not typically emitted.
-"""
+```python {class="line-numbers linkable-line-numbers"}
+my_input_controller = Controller.from_robot(robot=myRobotWithController, name="my_controller") ...
 
-POSITION_CHANGE_ABSOLUTE = "PositionChangeAbs"
-"""
-Absolute position is reported via Value, a la joysticks.
-"""
+# Get the list of Controls provided by the controller.
+controls = await my_input_controller.get_controls()
 
-POSITION_CHANGE_RELATIVE = "PositionChangeRel"
-"""
-Relative position is reported via Value, a la mice, or simulating axes with up/down buttons.
-"""
+# Print the list of Controls provided by the controller.
+print(f'Controls:\n{controls}')
 ```
 
-See [the Python SDK Docs](https://python.viam.dev/autoapi/viam/components/input/input/index.html#viam.components.input.EventType) for the most current version of supported `EventTypes`.
-
 {{% /tab %}}
 {{% tab name="Go" %}}
 
-``` go {class="line-numbers linkable-line-numbers"}
- // Callbacks registered for this event will be called in ADDITION to other registered event callbacks.
-AllEvents EventType = "AllEvents"
+**Parameters:**
 
-// Sent at controller initialization, and on reconnects.
-Connect EventType = "Connect"
+- `Context` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
 
-// If unplugged, or wireless/network times out.
-Disconnect EventType = "Disconnect"
+**Returns:**
 
-// Typical key press.
-ButtonPress EventType = "ButtonPress"
+- `controls` [([]float64)](https://pkg.go.dev/builtin#float64): List of controls provided by the controller.
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
 
-// Key release.
-ButtonRelease EventType = "ButtonRelease"
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/input#Controller).
 
-// Key is held down. This will likely be a repeated event.
-ButtonHold EventType = "ButtonHold"
+```go {class="line-numbers linkable-line-numbers"}
+myController, err := controller.FromRobot(myRobotWithController, "my_controller")
```

Comment:
why is it myController instead of myInputController if movement sensor is being expanded from my_sensor to my_movement_sensor in the other doc?

Reply:
okay fine as is then!